recipe/vina.patch

```diff
@@ -1,50 +1,39 @@
-From bfb38b20c80b9d44f9f4ecb822a2420238013dce Mon Sep 17 00:00:00 2001
-From: Conda Forge
-Date: Tue, 5 Oct 2021 14:23:12 +0200
-Subject: [PATCH] vina patch
```

Comment:
Please keep the patch format; generate it as `git format-patch <tag_for_1.2.5> --no-signature`